src/main/java/uk/gov/companieshouse/web/accounts/controller/govuk/GovukCorporationTaxController.java

```diff
@@ -22,8 +24,11 @@
     @GetMapping
     public String getCorporationTax(Model model) {
 
+        EnvironmentReader enviromentReader = new EnvironmentReaderImpl();
+
         model.addAttribute("corporationTax", new CorporationTax());
         model.addAttribute("hideUserBar", true);
+        model.addAttribute("backButton", enviromentReader.getMandatoryString("GOV_UK_FULL_ACCOUNTS_URL"));
```

Comment:
Could we make `"GOV_UK_FULL_ACCOUNTS_URL"` a constant rather than using a sting literal here?

Reply:
This has changed now as we are getting the value from the application.properties file to be consistent with the other external urls used in the application.